src/components/body/content/videoplayer/Videoplayer.tsx

```diff
@@ -0,0 +1,206 @@
+import React from "react";
+import Plyr from "plyr-react";
+// import Vimeo from '@u-wave/react-vimeo';
+import "plyr-react/plyr.css";
+
+import './index.scss';
+
+const videoSrc  = {
+  type: "video" as const,
+  title: "Elephants",
+  sources: [
+    {
+      src:
+        "https://rawcdn.githack.com/chintan9/Big-Buck-Bunny/915c4b2aba75614b20dec3852375b394bb305f10/ElephantsDream.mp4",
+      type: "video/mp4",
+      size: 576,
+    },
+    {
+      src:
+        "https://rawcdn.githack.com/chintan9/Big-Buck-Bunny/915c4b2aba75614b20dec3852375b394bb305f10/ElephantsDream.mp4",
+      type: "video/mp4",
+      size: 720,
+    },
+    {
+      src:
+        "https://rawcdn.githack.com/chintan9/Big-Buck-Bunny/915c4b2aba75614b20dec3852375b394bb305f10/ElephantsDream.mp4",
+      type: "video/mp4",
+      size: 1080,
+    },
+
+  ],
+  poster: 'https://upload.wikimedia.org/wikipedia/commons/thumb/e/e4/Elephants_Dream_cover.jpg/1200px-Elephants_Dream_cover.jpg?20060831021346',
+  tracks: [
+    {
+      kind: "captions" as const,
+      label: "Russian",
+      srclang: "ru",
+      src: "../subtitles/subtitles-ru.vtt",
+      default: true
+    },
+    {
+      kind: "captions" as const,
+      label: "English",
+      srclang: "en",
+      src: "../subtitles/subtitles-en.vtt",
+      default: true
+    },
+  ],
+  previewThumbnails: {
+    enabled: true,
+    src: ['https://cdn.plyr.io/static/demo/thumbs/100p.vtt', 'https://cdn.plyr.io/static/demo/thumbs/240p.vtt'],
+  },
+  tooltips: {
+    controls: true,
+  },
+  //Another exemple  a videoplayer
```

Comment:
Please remove comments - your code looks dirty. If you need working examples save them as javascript object in separate file or just here. Then simple switch them as props or something like this

Reply:
corrected